app/views/events/_new.html.erb

```diff
@@ -14,12 +14,12 @@
 
             <div class="form-group">
               <%= f.label :start_on, "Start date", class: "control-label" %>
-              <%= f.date_field :start_on, class: "form-control" %>
+              <%= f.text_field :start_on, class: "form-control datepicker", placeholder: "dd/mm/yyyy" %>
```

Comment:
Let's [remove the placeholders](https://www.nngroup.com/articles/form-design-placeholders/). 😀